Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN apk add --update --no-cache --virtual runtime-dependances \
  yarn \
  openssl-dev
 
+# Remove once base image ruby:2.7.5-alpine3.15 has been updated with latest gmp
```

Comment:
Is it worth putting another comment at the top next to the FROM line so we don't miss this when upgrading the base image?

Reply:
done